README.md

```diff
@@ -22,3 +22,25 @@ Tested on Ubuntu 18.04 and Debian 10.
 * Put all these files in `/var/www/html`
 * Run `sudo setup.sh`
 * Visit http://localhost in your browser
+
+FAQ
+----
+**Can you delete it when you are done with it?**
+
+Yes. For any wiki you create, you will see a `Delete` link in the `Action` column of this table: https://patchdemo.wmflabs.org/
```

Comment:
I would add that people **should** delete wikis when they are done, or the patch is merged.

Reply:
Done.